osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -99,7 +100,9 @@ private void setDistances(double clockRate)
                 // For hitobjects which continue in the direction of the slider, the player will normally follow through the slider,
                 // such that they're not jumping from the lazy position but rather from very close to (or the end of) the slider.
                 // In such cases, a leniency is applied by also considering the jump distance from the tail of the slider, and taking the minimum jump distance.
-                MovementDistance = Math.Min(JumpDistance, tailJumpDistance);
+                // Additional distance is removed based on position of jump relative to slider follow circle radius.
+                // JumpDistance is the distance beyond the s. tailJumpDistance is maximum_slider_radius since the full distance of radial leniency is still possible.
```

Comment:
what's s.? slider? please expand

Reply:
Resolved in base PR (https://github.com/ppy/osu/pull/14846).